src/plugin/v2/grpc_plugin.ts

```diff
@@ -137,6 +137,11 @@ class PluginGRPCServer {
       headers: headers,
     };
 
+    // Encoding parameter is not send over grpc
+    if (req.url.includes("encoding=pdf")) {
+      options.encoding = "pdf"
+    }
```

Comment:
What's the benefit of doing this compared to adding this field in the GRPC request object?

Reply:
I would flip the question -- what is the benefit of encoding it in gRPC?

From what I see, we are sending the encoding parameter in the URL regardless -- but honestly unclear how it is used in other contexts

Reply:
the main downside I see is that we would also have to send our format in the http flavor -- I suppose next to encoding?

Reply:
First, having `encoding` as a gRPC parameter would be more consistent with the server mode that already reads the `encoding` option from the request parameters, not the URL ones. 
It's also a cleaner solution as this will be clearly defined as a parameter in the proto files, rather than in the URL where nothing is defined/documented. Parameters in the URL should only be used by Grafana when the image renderer is accessing the URL through Chrome. I understand your choice for the PDF options, and I think it's OK to let them live in the URL as Grafana will also use them, but I don't think it's the case for `encoding`. 

Also, you did your tests with the `/render` endpoint but the goal is to use this in the reporting feature where it won't be in the URL.